llvm/lib/Transforms/Yk/ModuleClone.cpp

```diff
@@ -126,30 +126,60 @@ struct YkModuleClone : public ModulePass {
   }
 
   /**
-   * @brief Updates call instructions in cloned functions to reference
-   * other cloned functions.
+   * @brief Helper function to update call instructions within a function
```

Comment:
I believe LLVM prefers `///` for docstrings. See https://llvm.org/docs/CodingStandards.html#comment-formatting

Reply:
May I also suggest rewording this slightly to something like: "Replace all function calls within function `F` with calls to (unoptimised?/optimised?) functions, if such a function exists."